### PR TITLE
DEV-55 Setup postgresql connection

### PR DESCRIPTION
**Pull request by s-kakol**
Changes overview:

- `Dockerfile` - env vars decrease image size without impacting functionality
- `Docker-compose.yml` - add dependency on **db** service and env variables for the DB
- `.gitignore` - remove duplicated text & ignore _jetbrains_ .idea directory
- `backend/settings.py` - modify databases config to use postgresql
- `README.md` - expand upon existing doc

### CODE REVIEW

**Review by matgrube — APPROVED** [review]
Good stuff!

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.venv
+.git
+.gitignore
```
Comment (adjurk) [inline .dockerignore]: You can shorten this to: `.git*`. Good idea nonetheless!

File: README.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
-# Blood Donor - Django Backend
+# Blood Donor - Server
 
 The main REST API backend responsible for serving stateful data for the Blood Donor mobile apps.
 
-# Building & Running
-
-If you're building this project for the first time, you'll need [Docker](https://docs.docker.com/get-docker/).
+### Building & Running
```
Comment (adjurk) [inline README.md]: According to [markdownlint - MD001 - Heading levels should only increment by one level at a time](https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md), you should decrease the heading level to `##`.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+This package uses Python and Docker, make sure you've got that installed. \
+Required environment variables template available in ***.env.example*** file.
```
Comment (adjurk) [inline README.md]: Filenames should be enclosed with inline code: `.env.example`

**Review by adjurk — APPROVED** [review]
Apart from the formatting stuff, these changes are appreciated!

**Review by adjurk — APPROVED** [review]
LGTM now, thx!
